challenge.md

```diff
@@ -32,19 +32,19 @@ Open up this file in your text editor, and write your answers below (where indic
 1. Is `/Users/blah_blah/Desktop` a relative path or an absolute path? How do you know?
 
  <!-- Answer Starts Here -->
- 
+ It is a relative path because of the /
  <!-- Answer Ends Here -->
 
 2. Suppose that we're working on a project, and we want to use a font that's being hosted somewhere on the internet. Would we use an absolute or relative path to refer to it? Why?
 
  <!-- Answer Starts Here -->
- 
+ An absolute path becuase we want to refer to the location of the file and directories within the file system.
  <!-- Answer Ends Here -->
 
 3. Now suppose that we have an image file living inside our project. Would we refer to it with an absolute or relative path? Why?
 
  <!-- Answer Starts Here -->
- 
+ We would refer to it with a relative path because it is relative to the root directory.
```

Comment:
We would refer to it with a relative path, but not for this reason. Imagine that we had a project where all of our links used Absolute paths. Suppose that you zipped up that project and gave it to someone else - say, me. When I open the project, would any of those links work? No, because all of those links would refer to locations on _your_ computer. As a result, all of the links would be broken. However, the files within the project are still in the same place _relative to each other_, so if we'd used Relative paths, those links would still work.

As a rule of thumb, linking to a resource within the same project means using a Relative path, while linking to a resource outside your project means using an Absolute path.

As a rule of thumb, if you're linking to a resource _within your project_, use a relative